src/contexts/language/LanguageContext.ts

```diff
@@ -48,4 +48,4 @@ export class LanguageContext extends ContextBase {
   getProjectComponentContext(component: ProjectComponent): ProjectComponentContext {
     return this.projecComponentContextFactory(component);
   }
-}
+}
```

Comment:
add a new line at end of a file
```suggestion
}

```